string camera_id
AprilTagDetection[] detections
